docs/HowTo/Monitor/Quorum-Hibernate.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Use Quorum Hibernate with GoQuorum
+---
+
+# Quorum Hibernate
+
+[Quorum Hibernate] is a proxy that monitors a node's API traffic and hibernates the node when
+inactive. This reduces infrastructure costs by ensuring only nodes receiving API requests are
```

Comment:
Maybe adding a word like about the fact that validator nodes necessary for the consensus to establish can not be stopped.

e.g

This reduces infrastructure costs by ensuring only nodes receiving API requests or validator nodes necessary for the consensus to establish are running.

Reply:
Done